src/main/features/dashboard/views/macro/caseStatus.njk

```diff
@@ -47,45 +48,45 @@
     { ccjRequestedAt: claim.countyCourtJudgmentRequestedAt | date }) }}
   {% elseif status === ClaimStatus.ELIGIBLE_FOR_CCJ %}
-    <p>{{ t('You can request a County Court Judgment') }}</p>
+    <p>{{ t('The defendant has not responded to your claim. You can request a County Court Judgment against them.') }}</p>
   {% elseif status === ClaimStatus.OFFER_SETTLEMENT_REACHED %}
-    {{ t('You both signed an agreement to settle out of court.') }}
-    <p>
-      <a href="{{ OfferPaths.agreementReceiver.evaluateUri({ externalId: claim.externalId }) }}">{{ t('Download the agreement') }}</a>
-    </p>
+    {{ t('You’ve both signed a legal agreement. The claim is now settled.') }}
   {% elseif status === ClaimStatus.CLAIM_REJECTED %}
     {% if claim.response.defenceType === DefenceType.ALREADY_PAID %}
-      {{ t('The defendant believes that they’ve paid the claim in full.') }}
+        {{ t('{{ defendantName }} believes that they’ve paid the claim in full. ',
+        { defendantName: defendantName }) }}
     {% else %}
       {% if claim.response.freeMediation === FreeMediationOption.YES %}
         {{ t('{{ defendantName }} has rejected the claim. '
           + 'They’ve suggested free mediation on {{ respondedAt }}. You can accept or reject',
           { defendantName: defendantName, respondedAt: claim.respondedAt | date }) }}
       {% else %}
-        {{ t('{{ defendantName }} has rejected the claim. The case will be reviewed by a judge and might go to court.',
+        {{ t('{{ defendantName }} has rejected your claim.',
         { defendantName:  defendantName }) }}
       {% endif %}
     {% endif %}
-    <p>{{ _viewResponseLink(claim.externalId) }}</p>
   {% elseif status === ClaimStatus.OFFER_ACCEPTED %}
     {{ t('You’ve signed the agreement. We’ve asked {{ defendantName }} to sign the agreement.',
     { defendantName: defendantName }) }}
   {% elseif status === ClaimStatus.OFFER_REJECTED %}
     {{ t('You’ve rejected an offer to settle out of court.') }}
+    <p>
+      {{ t('{{ defendantName }} has rejected your claim.',
+      { defendantName:  claim.claimData.defendant.name }) }}
+    </p>
   {% elseif status === ClaimStatus.OFFER_SUBMITTED %}
-    {{ t('The defendant has made an offer to settle out of court.') }}
     <p>
-      <a href="{{ url }}">{{ t('View their offer') }}</a>
+      {{ t('The defendant has made an offer to settle out of court.') }}
     </p>
+    {{ t('{{ defendantName }} has rejected your claim.',
+    { defendantName:  claim.claimData.defendant.name }) }}
   {% elseif status === ClaimStatus.MORE_TIME_REQUESTED %}
-    {{ t('{{ defendantName }} has requested an extra 14 days to respond. They need to respond by {{ responseDeadline }}.',
-    { defendantName: defendantName, responseDeadline: claim.responseDeadline | date }) }}
+    {{ t('{{ defendantName }} has requested more time to respond.',
+    { defendantName: claim.claimData.defendant.name }) }}
```

Comment:
this can be changed to { defendantName:  defendantName }) ??!!

Reply:
All addressed in the recent push 👍